File: 이진영/BruteForce/카펫.js
function solution(brown, yellow) {
    let sum = brown + yellow;
    
    for (let i = 3; i <= sum / 3; i++) {
        // i는 세로로 정의
        if (sum % i === 0) {
            let j = sum / i; // 가로
            
            if (2 * i + 2 * j - 4 === brown) return [j, i]
        }
    }
    return [-1, -1]
}

File: 김채린/BruteForce/카펫.kt
class Solution {
    fun solution(
        brown: Int,
        yellow: Int,
    ): IntArray {
        var h = 0
        val result: IntArray
        while (true) {
            h++
            if (yellow % h!=0){continue}
            val nowW = yellow / h
            val nowBrown = (nowW + 2) * 2 + h * 2
            if (nowBrown == brown) {
                result = intArrayOf(nowW + 2, h + 2)
                break
            }
        }
        return result
    }
}
